tests/msim/MSIM_PYACTION_INSERT_KEYWORD.DATA

```diff
@@ -0,0 +1,541 @@
+-- This reservoir simulation deck is made available under the Open Database
```

Comment:
Where does this file come from? Please add a comment about this somewhere below.
Please also include comments if you changed something.